activemq-broker/src/main/java/org/apache/activemq/broker/region/DestinationStatistics.java

```diff
@@ -233,6 +222,7 @@ public void setParent(DestinationStatistics parent) {
             maxUncommittedExceededCount.setParent(parent.maxUncommittedExceededCount);
             networkEnqueues.setParent(parent.networkEnqueues);
             networkDequeues.setParent(parent.networkDequeues);
+            // [AMQ-9437] Advanced Message Statistics do not parent.
```

Comment:
```suggestion
            // [AMQ-9437] Advanced Message Statistics does not have a parent.
```

Reply:
Changes to comment applied via commit c079d1bd03df5a2faf35f2ec9625e7e51b2a4973